source/Views/DetailsViewGameOverview.xaml

```diff
@@ -123,7 +123,13 @@
 
                                 <StackPanel Grid.Row="0" Name="DetailsTop" VerticalAlignment="Top">
                                     <Grid Name="HeaderImagesGrid" Height="{Settings CalculatedGameDetailsIndentation}">
-                                        <ContentControl x:Name="ExtraMetadataLoader_LogoLoaderControl" Margin="20,20,30,20"/>
+                                        <ContentControl x:Name="ExtraMetadataLoader_LogoLoaderControl" Margin="20,20,30,20">
+											<ContentControl.Style>
```

Comment:
Formatting is broken here. Make sure the closing `</ContentControl>` and everything else is aligned. Also I think it's not needed to create a style for the control, you can set the visibility in the ContentControl opening properties.